### PR TITLE
[ENH][12.0]Edit create_edit in Lot/Serial Number

### PR DESCRIPTION
**Pull request by kranokporn**
@newtratip

### CODE REVIEW

File: rjc_stock/views/stock_move_views.xml [hunk 1]
```diff
@@ -7,7 +7,9 @@
         <field name="arch" type="xml">
             <field name="lot_id" position="attributes">
                 <attribute name="options">{'create': True}</attribute>
+                <attribute name="options">{'create_edit': False}</attribute>
             </field>
         </field>
     </record>
+
```
Comment (newtratip) [inline rjc_stock/views/stock_move_views.xml]: Remove this line

File: rjc_stock/views/stock_move_views.xml [hunk 2]
```diff
@@ -7,7 +7,9 @@
         <field name="arch" type="xml">
             <field name="lot_id" position="attributes">
                 <attribute name="options">{'create': True}</attribute>
+                <attribute name="options">{'create_edit': False}</attribute>
```
Comment (newtratip) [inline rjc_stock/views/stock_move_views.xml]: If you comment `<attribute name="options">{'create': True}</attribute>` out, Is that OK ?

You don't need to add `<attribute name="options">{'create_edit': False}</attribute>`

**Review by newtratip — CHANGES_REQUESTED** [review]
Some review.

File: rjc_stock/views/stock_move_views.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
         <field name="inherit_id" ref="stock.view_stock_move_line_operation_tree"/>
         <field name="arch" type="xml">
             <field name="lot_id" position="attributes">
-                <attribute name="options">{'create': True}</attribute>
+                <attribute name="options">{'create': False}</attribute>
```
Comment (newtratip) [inline rjc_stock/views/stock_move_views.xml:9]: @kranokporn  This is not show create/edit in dropdown many2one field, Am I correct ?

Comment (kranokporn) [reply rjc_stock/views/stock_move_views.xml:9]: @newtratip I think that's right, this is not show in lot_id field. Option is boolean false.

Comment (newtratip) [reply rjc_stock/views/stock_move_views.xml:9]: OK

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Comment by newtratip** [comment]
@Saran440 Can you merge this ?

**Pull request merged**